release/dist/cli/cli.go

```diff
@@ -142,3 +151,18 @@ func runBuild(ctx context.Context, filters []string, targets []dist.Target) erro
 	fmt.Println("Done! Took", time.Since(st))
 	return nil
 }
+
+func parseSigningKey(path string) (crypto.Signer, error) {
+	if path == "" {
+		return nil, nil
+	}
+	raw, err := os.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+	b, _ := pem.Decode(raw)
```

Comment:
In the interest of eye-watering tightness, maybe check that the `[]rest` return value is 0 bytes long? If there was extra stuff at the end of the PEM file, it's possible something weird or bad is happening.